tests/main/snapd-homedirs/task.yaml

```diff
@@ -13,9 +13,7 @@ prepare: |
     # Create a new user in a non-standard location
     mkdir -p /remote/users
     useradd -b /remote/users -m -U "$USERNAME"
-
-    # Install our test snap
-    "$TESTSTOOLS"/snaps-state install-local test-snapd-sh
+    snap install test-snapd-sh
```

Comment:
minor question, Is there a specific reason to use the store version instead of the local snap?

Reply:
sorry I should have been clearer on why I used the store version. The main reason was because the store version had multiple revision numbers tied to it which allowed me to do the following:
```
echo "When the snap is refreshed"
snap refresh --channel=edge test-snapd-sh
new_rev=$(snap list test-snapd-sh|tail -n1|tr -s ' '|cut -f3 -d' ')
```
which was needed to test the data copy/delete between revisions

Reply:
makes sense, thanks!